app/models/notification.rb

```diff
@@ -23,4 +23,9 @@ class Notification < ApplicationRecord
   def read?
     self.read_at
   end
+
+  def mark_as_read(time = Time.now)
```

Comment:
what do you think of calling this method `read!`?

Reply:
I prefer this name as well - it fits better with ruby conventions.

Reply:
Just note that #308 makes heavy use of this method and it's not merged yet - @aapomm , do you support this change?